lib/al/Library/Nerve/NerveStateBase.h

```diff
@@ -11,22 +11,22 @@ class NerveStateBase : public al::NerveExecutor {
 
 public:
     NerveStateBase(const char* stateName);
-    ~NerveStateBase();
+    inline ~NerveStateBase();
```

Comment:
i didn't try it on everything. it didn't change anything for one of my destructors though, so i'll probably just remove it just to keep everything the same.

Reply:
You should put the function body in the header (also you don't need the inline keyword for classes usually), this is probably what makes the PlayerStateGroundSpin dtor match, I saw the diff when you were working on it and it was calling ~NerveStateBase instead of ~NerveExecutor directly.